src/Microsoft.ML.Transforms/Text/TextCatalog.cs

```diff
@@ -283,7 +283,7 @@ public static class TextCatalog
             int maximumNgramsCount = NgramExtractingEstimator.Defaults.MaximumNgramsCount,
             NgramExtractingEstimator.WeightingCriteria weighting = NgramExtractingEstimator.WeightingCriteria.Tf)
             => new WordBagEstimator(Contracts.CheckRef(catalog, nameof(catalog)).GetEnvironment(),
-                outputColumnName, inputColumnName, ngramLength, skipLength, useAllLengths, maximumNgramsCount);
+                outputColumnName, inputColumnName, ngramLength, skipLength, useAllLengths, maximumNgramsCount, weighting);
```

Comment:
>weighting [](start = 111, length = 9)

Would be nice to add baseline test, something similar to `TestWordEmbeddings`.
Thank you for catching and fixing it!